tests/rental.spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../rental'
+
```

Comment:
- [ ] As mentioned by the previous reviewer and ad required for the project, kindly make sure  all the test files are placed in a directory named `spec` and not `tests` i.e. you can only edit the `tests` to `spec`